pkg/utils/kube/cert.go

```diff
@@ -0,0 +1,30 @@
+package kube
+
+import (
+	"context"
+	"crypto/x509"
+	"fmt"
+
+	"github.com/kyverno/kyverno/pkg/config"
+	"google.golang.org/grpc/credentials"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func FetchCert(
+	ctx context.Context,
+	certs string,
+	kubeClient kubernetes.Interface) (credentials.TransportCredentials, error) {
+	secret, err := kubeClient.CoreV1().Secrets(config.KyvernoNamespace()).Get(ctx, certs, v1.GetOptions{})
```

Comment:
What's the frequency of this client call?

Reply:
Just one time when kyverno is first started (or restarted...)